AVLTree/main.c

```diff
@@ -0,0 +1,93 @@
+#include <stdbool.h>
+#include "tree.h"
+#include <stdio.h>
+#include <locale.h>
+#include <stdlib.h>
+
+bool test(void);
+
+enum options { add = 1, get, check, del };
+
+void clearBuffer() {
+    char c;
+    while ((c = getchar()) != '\n' && c != EOF && c != NULL);
+}
+
+void chooseOption(int choice, Node** root) {
+    switch (choice) {
+    case add: {
+        char* key = calloc(150, 1);
```

Comment:
```suggestion
        char* key = calloc(150, sizeof(char));
```
Однако почему именно 150? И зачем тут аллокация? Всё равно было бы разумно копировать значения при добавлении в дерево, и тогда буферы ввода можно было бы разместить на стеке.